src/pal/src/misc/fmtmessage.cpp

```diff
@@ -51,11 +51,15 @@ typedef VOID * HSATELLITE;
 static HSATELLITE s_hSatellite = NULL;
 static LPVOID s_lpLibRotorPalRt = NULL;
 typedef HSATELLITE (__stdcall *FnLoadSatelliteResource)(LPCWSTR);
-static FnLoadSatelliteResource LoadSatelliteResource;
 typedef BOOL (__stdcall *FnFreeSatelliteResource)(HSATELLITE);
 static FnFreeSatelliteResource FreeSatelliteResource;
+#ifndef FEATURE_PAL
```

Comment:
The FEATURE_PAL ifdefs here do not make a sense ... this file is always going to be compiled with FEATURE_PAL defined.